mapbox_vector_tile/encoder.py

```diff
@@ -46,34 +48,36 @@ def addFeatures(self, features, layer_name=""):
             if geometry_spec is None:
                 continue
             shape = self._load_geometry(geometry_spec)
+
             if shape is None:
                 raise NotImplementedError(
                     'Can\'t do geometries that are not wkt, wkb, or shapely '
                     'geometries')
+
             if shape.is_empty:
                 continue
-            # if we are a multipolygon, we will create separate
-            # features for each polygon and duplicate properties
-            if shape.type == 'MultiPolygon':
-                exploded_features = self.explode_multipolygon_features(
-                    feature, shape)
-                for exploded_feature in exploded_features:
-                    self.addFeature(exploded_feature,
-                                    exploded_feature['geometry'])
-            else:
-                self.addFeature(feature, shape)
 
-    def explode_multipolygon_features(self, feature, shape):
+            if shape.type == 'MultiPolygon':
```

Comment:
I worry that if we don't enforce it, then a client library might interpret an inner as an outer and we'd start losing holes.